### PR TITLE
PR  [updated README with BEM]

### PR DESCRIPTION
**Pull request by tissakhosla**

### CODE REVIEW

File: client/README.md [hunk 1]
```diff
@@ -62,3 +62,32 @@ To generate a **class** component, run the following
 ```sh
 hygen component-class new --name NameOfComponent
 ```
+### CSS Class Names
+
+- To avoid style conflicts we want to make sure all our CSS selectors are unique.
+- This will be the naming convention, based on BEM. [https://css-tricks.com/bem-101/]
```
Comment (ZakkMan) [inline client/README.md]: Can you turn this into a Markdown link?

File: client/README.md [hunk 2]
```diff
@@ -62,3 +62,32 @@ To generate a **class** component, run the following
...
+```
+.ComponentName
+.ComponentName--modifier
+.ComponentName__elementName
+.ComponentName__elementName--modifier
```
Comment (ZakkMan) [inline client/README.md]: Did y'all decide on `elementName` over `element-name`? I think the PR I merged is using `element-name`

Comment (wchrapcynski) [reply client/README.md]: Just a note. I personally like element-name and keeping the camel case for JS just to have things separate.

File: client/README.md (hunk 1 above)
Comment (ZakkMan) [reply client/README.md]: @tissakhosla did you miss this one?

**Pull request merged**